docs/Clearwater_Configuration_Options_Reference.md

```diff
@@ -117,7 +117,7 @@ This section describes optional configuration options, particularly for ensuring
 * `sip_blacklist_duration` - the time in seconds for which SIP peers are blacklisted when they are unresponsive (defaults to 30 seconds).
 * `http_blacklist_duration` - the time in seconds for which HTTP peers are blacklisted when they are unresponsive (defaults to 30 seconds).
 * `diameter_blacklist_duration` - the time in seconds for which Diameter peers are blacklisted when they are unresponsive (defaults to 30 seconds).
-* `snmp_ip` - the IP address to send alarms to (defaults to being unset). If this is set then Sprout, Ralf, Homestead and Chronos will send alarms - more details on the alarms are [here](http://clearwater.readthedocs.org/en/stable/SNMP_Alarms/index.html)
+* `snmp_ip` - the IP address to send alarms to (defaults to being unset). If this is set then Sprout, Ralf, Homestead and Chronos will send alarms - more details on the alarms are [here](http://clearwater.readthedocs.org/en/stable/SNMP_Alarms/index.html). This can be a single IP address, DNS A record name, or a comma-separated list of IP addresses.
```

Comment:
If the DNS A records resolves to multiple IP addresses, do we send SNMP traps to one, all, or some of them... and do we chose the same one/some for each trap or are they load-balanced?